mesh/activation.go

```diff
@@ -0,0 +1,100 @@
+package mesh
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/davecgh/go-xdr/xdr2"
+	"github.com/spacemeshos/go-spacemesh/common"
+	"github.com/spacemeshos/go-spacemesh/crypto"
+	"github.com/spacemeshos/go-spacemesh/nipst"
+	"io"
+)
+
+//todo: choose which type is VRF
+type Vrf string
+
+type Id struct {
```

Comment:
i think we should give this a more descriptive name.
mesh.Id can pretty much mean anything in our context